webpack/assets/javascripts/react_app/components/factCharts/FactChartSelectors.js

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect';
+
+const values = state => state.factChart.chartData;
+const hostCounter = (accumulator, currentValue) =>
+  accumulator + currentValue;
+
+export const selectHostCount = createSelector(
+  values,
+  chartData =>
+    (chartData.length > 0
+      ? chartData.map(item => item[1]).reduce(hostCounter)
+      : 0),
+);
+
+export const selectFactChart = state => state.factChart;
```

Comment:
1. Does it make sense to reuse the `selectFactChart` in `values`?
2. Does it make sense to export the `values` selector under `selectFactChartData`?

```js
export const selectFactChart = state => state.factChart;
export const values = state => selectFactChart(state).chartData;
```

I think it helps to have a single responsibility for each part of the state.